packages/terra-tabs/src/orion-fusion-theme/Tabs.module.scss

```diff
@@ -98,6 +98,12 @@
     --terra-tabs-menu-icon-margin-right: 0.25rem;
     --terra-tabs-menu-icon-font-size: 1rem;
 
+    --terra-tabs-addButton-focus-background-color: #fff;
+    --terra-tabs-addButton-focus-background-image: none;
```

Comment:
Not able to find the reference for background-image property in both default theme and clinical-light-theme

Reply:
Done https://github.com/cerner/terra-framework/commit/fcb1925802030dfb5be6bf821a0a978a5c82d40c